src/tls.c

```diff
@@ -10408,6 +10495,14 @@ void TLSX_FreeAll(TLSX* list, void* heap)
                 break;
 #endif
 
+    #ifdef WOLFSSL_QUIC
+            case TLSX_KEY_QUIC_TP_PARAMS:
+                FALL_THROUGH;
+            case TLSX_KEY_QUIC_TP_PARAMS_DRAFT:
+                QuicTransportParam_free((QuicTransportParam*)extension->data, heap);
```

Comment:
For consistency, add a define like QTP_FREE that maps to something.

Reply:
Added.